README.md

```diff
@@ -6,13 +6,13 @@ Sketch plugin to paste whatever image is on your clipboard as the fill for any s
 ## Installation
 
 1. Open Sketch and choose Plugins > Reveal Plugins Folder.
-2. Download this project and place the folder inside the Plugins directory.
+2. Download this project, open the folder, and double click the .sketchplugin file.
 
 
 ## Usage
 1.  Copy a screenshot to your clipboard **(shift + ctrl + cmd + 4)** or just right-click and copy image
 2. Select 1 or more shapes in Sketch
-3. Run the plugin **(ctrl + alt + v)**
+3. Run the plugin **(ctrl + alt + v)** or **(shift + cmd + v on a Mac)**
```

Comment:
This line is incorrect on Mac. The hotkey is ctrl + opt + v. As Sketch is only a Mac app, I'm curious where / how you're using it in a windows context?

Reply:
Whoops.  I meant on a "new mac keyboard", as they removed alt from the latest macs 😬. Will fix.